.azure-pipelines-templates/matrix.yml

```diff
@@ -119,17 +173,27 @@ jobs:
 
   # Publish release
   - ${{ if eq(parameters.perf_or_release, 'release') }}:
+    # Release build
+    - template: common_build.yml
+      parameters:
+        target: Build_SGX
+        env: ${{ parameters.env.Build_Release }}
+        cmake_args: '${{ parameters.build.common.cmake_args }} ${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }}'
+        suffix: 'Release'
+        artifact_name: 'Build_SGX_Release'
+
     # Release builds
     - ${{ each consensus in parameters.consensus }}:
-      - template: common.yml
+      - template: common_test.yml
         parameters:
           target: SGX
           consensus: ${{ consensus }}
           env: ${{ parameters.env.Release }}
           install_prefix: '${{ parameters.build.install.install_prefix }}'
-          cmake_args: '${{ parameters.build.common.cmake_args }} ${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }} ${{ parameters.build[consensus].cmake_args }}'
-          ctest_filter: '${{ parameters.test.Release.ctest_args }}'
+          cmake_args: '${{ parameters.build.common.cmake_args }} ${{ parameters.build.install.cmake_args }} ${{ parameters.build.SGX.cmake_args }}'
+          ctest_filter: '${{ parameters.test.Release[consensus].ctest_args }}'
           suffix: 'Release'
+          artifact_name: 'Build_SGX_Release'
```

Comment:
It looks like this won't work? e.g. `parameters.build.install.install_prefix` is not defined?